packages/os/os.spec

```
@@ -367,24 +548,53 @@ echo "** Output from non-static builds:"
     -p driverdog \
     %{nil}
 
+# Wait for fips builds from the background, if they're not already done.
+set +e; wait "${fips_pid}"; fips_rc="${?}"; set -e
+echo -e "\n** Output from FIPS builds:"
+cat "${fips_output}"
+
 # Wait for static builds from the background, if they're not already done.
 set +e; wait "${static_pid}"; static_rc="${?}"; set -e
 echo -e "\n** Output from static builds:"
 cat "${static_output}"
-if [ "${static_rc}" -ne 0 ]; then
-   exit "${static_rc}"
-fi
+
+set +e; wait "${static_fips_pid}"; static_fips_rc="${?}"; set -e
+echo -e "\n** Output from FIPS static builds:"
+cat "${static_fips_output}"
 
 # Wait for AWS SDK builds from the background, if they're not already done.
 set +e; wait "${aws_sdk_pid}"; aws_sdk_rc="${?}"; set -e
 echo -e "\n** Output from AWS SDK builds:"
 cat "${aws_sdk_output}"
+
+set +e; wait "${fips_aws_sdk_pid}"; fips_aws_sdk_rc="${?}"; set -e
+echo -e "\n** Output from AWS SDK FIPS builds:"
+cat "${fips_aws_sdk_output}"
+
+if [ "${fips_rc}" -ne 0 ]; then
+   exit "${fips_rc}"
+fi
+
+if [ "${fips_aws_sdk_rc}" -ne 0 ]; then
+   exit "${fips_aws_sdk_rc}"
+fi
+
+if [ "${static_fips_rc}" -ne 0 ]; then
+   exit "${static_fips_rc}"
+fi
+
+if [ "${static_rc}" -ne 0 ]; then
+   exit "${static_rc}"
+fi
```

is there any reason to change the behavior from failing fast to waiting for each background build to complete, then checking for build success/failure?

---

This guarantees that if multiple jobs failed we will get all the outputs. If bail in the first fail, we will only get the output for that failure.